### ISSUE TITLE
Any plan to release the code for transfer learning?

### ISSUE THREAD

**Issue by xwen99** [deepmind/detcon#6]
Hi, thank you for your nice work. 

I'm now trying to reproduce some transfer learning (e.g., det, semantic seg) results with the released checkpoints, the checkpoint is transferred to pytorch-style with `byol-convert` and evaluated with another pytorch-based codebase (detectron2/mmseg), but the results are a bit lower than those reported in the paper (e.g., two-point drop in VOC semantic seg).

So I wonder if you have any plan to release the official code for transfer learning? As this may greatly help to reproduce your results.

**Comment by olivierhenaff** [comment]
Hello, we've added a checkpoint for a BYOL-pretrained ResNet-50 to our GCS bucket (resnet50_byol_imagenet1k.npy). Could you run your fine-tuning script on this model, to check the relative difference between BYOL and DetCon_B?

**Comment by xwen99** [comment]
Hi Oliver,

Recently I reproduced BYOL and DetCon_B with the provided weights (ImageNet 1k epoch pre-training), the results are as follows:

<!DOCTYPE html>

Method | COCO det | COCO seg | VOC | Cityscapes
-- | -- | -- | -- | --
  | AP | AP | mIoU | mIoU
BYOL (reported) | 41.6 | 37.2 | 75.7 | 74.6
BYOL (reproduced) | 41.6 | 37.6 | 71.8 (-3.9) | 74.8
DetCon (reported) | 42.7 | 38.2 | 77.3 | 77.0
DetCon (reproduced) | 42.2 | 38.1 | 75.6 (-1.7) | 76.6

The results on COCO and Cityscapes are acceptable, indicating that the checkpoints are okay.
The only problem lies in VOC. My implementation strictly follows MoCo (1st image), and I re-checked the implementation details in BYOL (2nd image) and DetCon (3rd image), and it seems that both works vary in the transfer learning settings on VOC (including the learning rate and operation on the inference image), just wonder does the difference harm fair comparison?

![image](https://user-images.githubusercontent.com/48824317/169797867-9b53fa54-27b8-406b-89a1-78cd56a1765c.png)
![image](https://user-images.githubusercontent.com/48824317/169797922-d3bb1c10-5555-4fa0-8955-c92a2771867a.png)
![image](https://user-images.githubusercontent.com/48824317/169797944-cbb1942a-24dd-42ec-8e39-7ec3aae16706.png)
